server/index.js

```diff
@@ -8,6 +8,7 @@ const friends = require("./controllers/friends");
 const preferences = require("./controllers/preferences");
 const locations = require("./controllers/locations");
 const users = require("./controllers/users");
+const e = require('express');
```

Comment:
Don't we already require express on [line 3](https://github.com/Pacific-Blue-Ocean/bored-tour/blob/37a9c5bd29757d2b26c129c0a426248770dfcd9b/server/index.js#L3) of this file?

Reply:
Must've been an autocomplete. I'll take it out.